### ISSUE TITLE
Recent Nvidia GPU's using another type of lower level dithering

### ISSUE THREAD

**Issue by Jack154nsk** [Maassoft/ColorControl#335]
Hello,

I use Color Control to disable dithering on my RTX A4000, this tells the driver to disable dithering, this works and i verified it with a lossless capture card DVI2PCIe but it seems recent Nvidia GPU's have a lower level of dithering that Color Control is not able to disable not be detected through the GPU's DP output.  The RTX A4000 produces a much nosier image with colors that are different than if i connect my true 10bit monitor into my old K4200. It certainly seems there is another type of dithering going on, im having trouble isolating what part of the GPU is controlling this but maybe this is something to further look into as having more control over the GPU's dithering behavior would be very desirable since its not needed and causes problems with accuracy on a true 10bit SDR panel.

**Comment by JordansGithub** [comment]
Yeah im also noticing the same thing with my RTX 3070 Founders card, Color Control works but there certainly seems to be a second type of dithering these cards are using.

**Comment by Maassoft** [comment]
I don't know what Nvidia is doing under the hood for this. I can't find any other dither-related functions in the NvAPI, so it might be an undisclosed/proprietary function.
What happens if you set the 'Color accuracy mode' to Reference instead of Accurate? Assuming that is available for workstation GPU's. I've found out that toggling this color accuracy mode also forces the driver to read the registry settings, including the normal dither settings, without restarting the driver. Unfortunately I can't find an NvAPI-function to control this from code.
